transformers/resolution_resource.go

```diff
@@ -1,14 +1,31 @@
 package transformers
 
 import (
+	"fmt"
+
+	"github.com/companieshouse/chs.go/log"
 	"github.com/companieshouse/insolvency-api/models"
+	"github.com/companieshouse/insolvency-api/utils"
 )
 
 // ResolutionResourceRequestToDB transforms a resolution request to a dao model
-func ResolutionResourceRequestToDB(req *models.Resolution) *models.ResolutionResourceDao {
+func ResolutionResourceRequestToDB(req *models.Resolution, transactionID string) *models.ResolutionResourceDao {
+	etag, err := utils.GenerateEtag()
```

Comment:
There is a need for a check on etag before continue its value is needed in following lines.